GitHub integration with Azure Pipelines: embedding automated tests issue

Missing `jest `dependency for executing the unit test task in the Pipeline.
```
2020-08-25T14:53:15.0479716Z ========================== Starting Command Output ===========================
2020-08-25T14:53:15.0504987Z [command]/bin/bash --noprofile --norc /home/vsts/work/_temp/59626d58-328e-4c34-979d-f1e8545e7762.sh
2020-08-25T14:53:15.3123160Z 
2020-08-25T14:53:15.3124615Z > contoso-air-dev@0.0.0 test /home/vsts/work/1/s
2020-08-25T14:53:15.3125380Z > jest --testRegex=.*\.i?spec\.js$
2020-08-25T14:53:15.3125743Z 
2020-08-25T14:53:15.3153833Z sh: 1: jest: not found
2020-08-25T14:53:15.3154488Z npm ERR! Test failed.  See above for more details.
2020-08-25T14:53:15.3154945Z 
2020-08-25T14:53:15.3184877Z ##[error]Bash exited with code '1'.
```

--- a/labs/vstsextend/github-azurepipelines/readme.md
+++ b/labs/vstsextend/github-azurepipelines/readme.md
@@ -249,7 +249,9 @@ Now that we have our CI successfully built, it's time to deploy but how do we kn
        inputs:
          command: 'custom'
          customcommand: 'install --production'
-     - script: 'npm test'
+     - script: |
+         npm install
+         npm test
        displayName: 'Run unit tests'
        continueOnError: true
      - task: PublishTestResults@2

